cloudformation/memsub-promotions-cf.yaml

```diff
@@ -407,7 +240,7 @@ Resources:
   TopicSendEmail:
```

Comment:
We could just use the existing SNS topic and pass that in as a parameter (instead of just passing in the email address)?

Reply:
Sounds much better, do you want to come and show me how to achieve that in CFN YAML?!